src/utilities/types.ts

```diff
@@ -133,6 +149,17 @@ export const TYPES: { [key: string]: Partial<IFieldConfig> } = {
     render: stripFieldConfig(formatPercentage),
     toForm: (data: any, field: string) => getPercentDisplay(get(data, field)),
   },
+  phone: {
+    editComponent: Antd.Input,
+    formValidationRules: {
+      isPhoneNumber: {
+        message: 'Must be a valid phone number',
+        pattern: /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/,
```

Comment:
I kinda like that you can see the entire field definition, including validation, in one place? But I'm fine pulling regex out. Maybe into consts?

Reply:
I agree with the entire field definition being there, including the validation rules. The only thing I suggest moving is the regex patterns here and below since regex is often 1. reused 2. tweaked slightly. Putting them in consts sounds good for now